### DISCUSSION TITLE
`vessel_id` should not be a mandatory field in `AbstractSpecies`

### DISCUSSION THREAD

**Discussion by haeussma** [EnzymeML/PyEnzyme#66] [General]
To my understanding the purpose of the different species definitions in EnzymeML is to add information on a species regardless of its context in a reaction. Thus, the mandatory link to a reaction vessel makes no sense imo. 
However, I think the link to the vessel is important and should be linked in the `Measurement` section. This would allow to define different vessels without the need to set up all species multiple times for each vessel.

Any opinions on that?

**Comment by JR-1991** [comment]
I see what you mean. We had to add it at the `Reactant` level to ensure we comply with the SBML requirements. So that you know, the `compartment_id` field of a `species` object is mandatory in SBML. On the other hand, we don't really use the `vessel_id` field that much in EnzymeML. If a `vessel_id` isn't provided, we could use a placeholder value instead.

Adding the vessel to a `Measurement` object is a good idea. However, a vessel could potentially also be used to store a certain substance or enzyme. Hence, I would still allow it to be added to the `AbstractSpecies` level.
